api/src/test/java/uk/gov/dhsc/htbhf/claimant/model/UpdatableClaimantFieldTest.java

```diff
@@ -44,7 +44,6 @@ void shouldReportFieldsDoNotHaveDifferentValues(UpdatableClaimantField field) {
     void shouldReportNullFieldsDoNotHaveDifferentValues(UpdatableClaimantField field) {
         Claimant originalClaimant = Claimant.builder().build();
         Claimant newClaimant = Claimant.builder().build();
-        assertThat(newClaimant).isNotEqualTo(originalClaimant);
 
         assertThat(field.valueIsDifferent(originalClaimant, newClaimant)).isFalse();
```

Comment:
Shouldn't we now be doing `assertThat(newClaimant).isEqualTo(originalClaimant);` I'm not sure that this assertion is any more helpful (ore required) than just using `isEqualTo`.